apps/builder/app/builder/features/sidebar-left/panels/pages/page-settings.tsx

```diff
@@ -607,6 +619,59 @@ const FormFields = ({
             </BindingControl>
           </Grid>
 
+          {isFeatureEnabled("cms") && (
+            <Grid gap={1}>
+              <Label htmlFor={fieldIds.status}>Status</Label>
+              <BindingControl>
+                <BindingPopover
+                  scope={scope}
+                  aliases={aliases}
+                  variant={
+                    isLiteralExpression(values.status) ? "default" : "bound"
+                  }
+                  value={values.status}
+                  onChange={(value) => {
+                    onChange({
+                      field: "status",
+                      value,
+                    });
+                  }}
+                  onRemove={(evaluatedValue) => {
+                    onChange({
+                      field: "status",
+                      value: JSON.stringify(evaluatedValue),
+                    });
+                  }}
+                />
+                <InputErrorsTooltip errors={errors.status}>
+                  <InputField
+                    inputMode="numeric"
+                    color={errors.status && "error"}
+                    id={fieldIds.status}
+                    name="status"
+                    placeholder="/another-path"
+                    disabled={
+                      disabled || isLiteralExpression(values.status) === false
+                    }
+                    value={String(
+                      computeExpression(values.status, variableValues)
+                    )}
+                    onChange={(event) => {
+                      const number = Number(event.target.value);
+                      const status = Number.isNaN(number)
```

Comment:
why do we try to convert it to number if we then stringify it right away?ˆ

Reply:
still not sure why we need to JSON stringify it

Reply:
200 -> '200'
"200text" -> '"200text"'

Reply:
200text is that a valid http status code?

Reply:
It's not. Validation will show the error